config/boot.rb

```diff
@@ -1,4 +1,5 @@
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../Gemfile", __dir__)
+ENV['EXECJS_RUNTIME'] = 'Node'
```

Comment:
do we need this?

Reply:
Deleted. Not sure how that got there lol